<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHola">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionGuardar"/>
    <addaction name="separator"/>
    <addaction name="actionSalir"/>
   </widget>
   <widget class="QMenu" name="menuReserva">
    <property name="title">
     <string>Reserva</string>
    </property>
    <addaction name="actionRealizar_una_reserva"/>
   </widget>
   <widget class="QMenu" name="menuAgregar_2">
    <property name="title">
     <string>Agregar</string>
    </property>
    <addaction name="actionJugador"/>
    <addaction name="actionCancha"/>
    <addaction name="actionEquipo"/>
    <addaction name="actionCliente"/>
   </widget>
   <widget class="QMenu" name="menuListado">
    <property name="title">
     <string>Listado</string>
    </property>
    <addaction name="actionJugadores"/>
    <addaction name="actionCanchas"/>
    <addaction name="actionEquipos"/>
    <addaction name="actionClientes"/>
   </widget>
   <addaction name="menuHola"/>
   <addaction name="menuReserva"/>
   <addaction name="menuAgregar_2"/>
   <addaction name="menuListado"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionAgregar_Jugador">
   <property name="text">
    <string>Agregar Jugador</string>
   </property>
  </action>
  <action name="actionAgregar_Cancha">
   <property name="text">
    <string>Agregar Cancha</string>
   </property>
  </action>
  <action name="actionAgregar_Equipo">
   <property name="text">
    <string>Agregar Equipo</string>
   </property>
  </action>
  <action name="actionRealizar_una_reserva">
   <property name="text">
    <string>Realizar una Reserva</string>
   </property>
  </action>
  <action name="actionEliminar_una_Reserva">
   <property name="text">
    <string>Eliminar una Reserva</string>
   </property>
  </action>
  <action name="actionGuardar">
   <property name="text">
    <string>Guardar</string>
   </property>
  </action>
  <action name="actionSalir">
   <property name="text">
    <string>Salir</string>
   </property>
  </action>
  <action name="actionJugador">
   <property name="text">
    <string>Jugador</string>
   </property>
  </action>
  <action name="actionCancha">
   <property name="text">
    <string>Cancha</string>
   </property>
  </action>
  <action name="actionEquipo">
   <property name="text">
    <string>Equipo</string>
   </property>
  </action>
  <action name="actionCliente">
   <property name="text">
    <string>Cliente</string>
   </property>
  </action>
  <action name="actionJugadores">
   <property name="text">
    <string>Jugadores</string>
   </property>
  </action>
  <action name="actionCanchas">
   <property name="text">
    <string>Canchas</string>
   </property>
  </action>
  <action name="actionEquipos">
   <property name="text">
    <string>Equipos</string>
   </property>
  </action>
  <action name="actionClienres">
   <property name="text">
    <string>Clientes</string>
   </property>
  </action>
  <action name="actionClientes">
   <property name="text">
    <string>Clientes</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
